test/plots/us-presidential-forecast-2016.js

```diff
@@ -0,0 +1,22 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/us-presidential-forecast-2016-histogram.csv", d3.autoType);
+  return Plot.plot({
+    x: {
+      label: 'Electoral votes for Hillary Clinton →',
+      labelAnchor: 'left',
```

Comment:
```suggestion
      label: "Electoral votes for Hillary Clinton →",
      labelAnchor: "left",
```

consistent style for quotes